packages/synapse-react-client/src/components/DataGrid/useDataGridWebsocket.ts

```diff
@@ -9,6 +9,7 @@ import { useDocumentVisibility } from '@react-hookz/web'
 interface WebSocketState {
   model: GridModel | null
   isGridReady: boolean
+  hasRenderableModel: boolean
```

Comment:
I'll want to rename `isGridReady` since it would have a different meaning with your changes.

Maybe something like 
`isGridReady` becomes: `hasCompletedInitialSync` or `isBackfilled`
`hasRenderableModel` could stay the same (I'm ok with the name you chose), but you could consider:  `isInteractive`, `hasSufficientData`, or `isGridReady`

At the very least, could you please add descriptive comments to at least these two properties in the `WebSocketState` type?